README.md

```diff
@@ -193,12 +199,18 @@ to update labels, milestones of PR from forked repositories.
 After approval or merged:
 - default label - `maintenance` will be added
 - current milestone will be set
- 
-## More options
 
-More options with default values can be found in workflow source in `inputs` section:
+## milestone configuration
 
-https://github.com/apache/maven-gh-actions-shared/blob/v4/.github/workflows/maven-verify.yml
+We need exactly **one** opened milestone in order to detect which is current.
+
+In case when we need more than one opened mielestone, we have add to milestone description:
```

Comment:
If we need more than one open milestone, we have to add a branch to the milestone description